src/app/tests/suites/certification/Test_TC_TCCM_3_1.yaml

```diff
@@ -21,183 +20,111 @@ PICS:
 
     - label:
-          "Step 5: TH sends a ChangeToMode command to the DUT with NewMode set
-          to new_mode_th"
+          "Select a value from supported_modes_dut different from on_mode_dut.
+          Save the value as new_mode_th. TH sends a ChangeToMode command to the
+          DUT with NewMode set to new_mode_th"
+      runIf: IsExpectedValue
       PICS: TCCM.S.C00.Rsp && TCCM.S.F00
-      verification: |
-          ./chip-tool refrigeratorandtemperaturecontrolledcabinetmode change-to-mode 2 1 1
-
-          Verify on TH(chip-tool) log, DUT responds contains a ChangeToModeResponse command with a SUCCESS (value 0x00) status response and below is the sample log provided for the raspi platform:
+      command: "ChangeToMode"
+      arguments:
+          values:
+              - name: "NewMode"
+                value: new_mode_th
 
-          [1690190379.801241][16449:16451] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0052 Command 0x0000_0001
-          [1690190379.801256][16449:16451] CHIP:TOO:   ChangeToModeResponse: {
-          [1690190379.801260][16449:16451] CHIP:TOO:     status: 0
-          [1690190379.801263][16449:16451] CHIP:TOO:    }
-      disabled: true
-
-    - label: "Step 6: TH sends a Off command to the DUT"
+    - label: "TH sends an Off command to the DUT"
       PICS: OO.S.C00.Rsp && TCCM.S.F00
-      verification: |
-          ./chip-tool onoff off 1 1
-
-          On TH(chip-tool) log,  Verify that DUT responds with a successful (value 0x00) status response and below is the sample log provided for the raspi platform:
+      cluster: "On/Off"
+      command: "Off"
 
-          [1684931101.118656][20709:20711] CHIP:DMG:                                 StatusIB =
-          [1684931101.118659][20709:20711] CHIP:DMG:                                 {
-          [1684931101.118663][20709:20711] CHIP:DMG:                                         status = 0x00 (SUCCESS),
-          [1684931101.118665][20709:20711] CHIP:DMG:                                 },
-      disabled: true
-
-    - label: "Step 7: TH sends a On command to the DUT"
+    - label: "TH sends an On command to the DUT"
       PICS: OO.S.C01.Rsp && TCCM.S.F00
-      verification: |
-          ./chip-tool onoff on 1 1
-
-          On TH(chip-tool) log, Verify that DUT responds with a successful (value 0x00) status response and below is the sample log provided for the raspi platform:
-
-          [1684931217.055514][20729:20731] CHIP:DMG:                                 StatusIB =
-          [1684931217.055517][20729:20731] CHIP:DMG:                                 {
-          [1684931217.055520][20729:20731] CHIP:DMG:                                         status = 0x00 (SUCCESS),
-          [1684931217.055523][20729:20731] CHIP:DMG:                                 },
-      disabled: true
-
-    - label: "Step 8: TH reads from the DUT the CurrentMode attribute."
+      cluster: "On/Off"
+      command: "On"
+
+    - label: "Wait to turn on/Off"
```

Comment:
This is not in the test plan.  Please at least add a comment explaining why it's here, but also file a test plan issue about this and point to it here.

Reply:
Removed it as it is not in test-plan.